src/App.tsx

```diff
@@ -1,9 +1,112 @@
+import { useState } from "react";
+import { useQuery } from "react-query";
+// Components
+import Item from "./components/Item/Item";
+import Cart from "./components/Cart/Cart";
+import Drawer from "@material-ui/core/Drawer";
+import LinearProgress from "@material-ui/core/LinearProgress";
+import Grid from "@material-ui/core/Grid";
+import AddShoppingCartIcon from "@material-ui/icons/AddShoppingCart";
+import Badge from "@material-ui/core/Badge";
+// Styles
+import { Wrapper, StyledButton } from "./App.styles";
+// Types
+export type CartItemType = {
+  id: number;
+  category: string;
+  description: string;
+  image: string;
+  price: number;
+  title: string;
+  amount: number;
+};
+
+const baseURL = "https://fakestoreapi.com/products";
+
+// await and await??
+const getProducts = async (): Promise<CartItemType[]> =>
+  await (await fetch(baseURL)).json();
```

Comment:
await fetch(baseURL) jest po to, zeby poczekac na wynik fetcha i dopiero po tym wywolac json,
a czemu jest przed tym jeszcze jeden to idk, moze zeby czekac na getProducts jak zostanie wywolane, ale wtedy to raczej przed wywolaniem getProduct sie daje await